client/sites-dashboard/utils.ts

```diff
@@ -55,6 +58,10 @@ export const isMigrationTrialSite = ( site: SiteExcerptNetworkData ) => {
 	return site?.plan?.product_slug === PLAN_MIGRATION_TRIAL_MONTHLY;
 };
 
+export const isECommerceTrialSite = ( site: SiteExcerptNetworkData ) => {
+	return site?.plan?.product_slug === PLAN_ECOMMERCE_TRIAL_MONTHLY;
+};
+
```

Comment:
If we use the existing selector, we can remove this function, what do you think?

Reply:
I would leave this function for future use since there is a similar one, `isMigrationTrialSite`.